great_expectations/data_context/types/base.py

```diff
@@ -680,15 +684,11 @@ def validate_schema(self, data, **kwargs):
 """
                 )
         if (
-            "data_asset_name_prefix" in data
```

Comment:
@lccasagrande I am not sure what the intent behind these deletions and additions are -- I am guessing that merging the latest `develop` into the present branch (and resolving any merge conflicts) might help, and I will be happy to look again at that time.

Reply:
Just a refactoring